### PR TITLE
ray assembler and rayplatform dependency

### PR DESCRIPTION
**Pull request by necrolyte2**

### CODE REVIEW

**Comment by necrolyte2** [comment]
Merge please

File: recipes/ray/build.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+ln -sf $PREFIX/lib/RayPlatform RayPlatform
+make
+cp -R Ray $PREFIX/bin/
+find scripts -type f \( -name "*.sh" -o -name "*.py" \) -exec cp {} $PREFIX/bin/ \;
```
Comment (tomkinsc) [inline recipes/ray/build.sh]: If `ray` is a pile of scripts (with some ancillary files), consider copying everything to a package-specific directory within `$PREFIX/share` or `$PREFIX/opt`, similar to [this recipe](https://github.com/bioconda/bioconda-recipes/blob/master/recipes/trimmomatic/build.sh) for trimmomatic. Doing so is more in line with the [Linux filesystem hierarchy](https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard), and it helps avoid collisions. You can symlink scripts to `$PREFIX/bin` from there.

**Comment by tomkinsc** [comment]
You might split the two recipes in this PR into separate pull requests, one for rayplatform and one for ray.

**Comment by druvus** [comment]
ray is now included in bioconda so closing this.

**Pull request closed without merging**